docs/minutes/cpansec-minutes-2024-07-03.md

```diff
@@ -0,0 +1,93 @@
+
+This meeting was done in Element/[Matrix], which uses Jitsi. Went well.
+
+### CPANminus
+1. We have sent PRs to master and development
+2. An independant PR has been sent into the official perl-docker image.
+3. Trying to push for use of the new patched version
+4. What are the next steps 
+5. Some arguement about whether to accept into the docker images.
+6. @Tux mentioned the failback is needed - an environment variable would suffice
+
+### CNA process is initiated with Mitre
+1. @timlegge: Mitre finally got back to me and has some information they need as well as scheduling a meeting with them to discuss
+2. @timlegge Stig and I are discussing
+3. We could use some additional help as we go further with a CNA  Please reach out if you have time and an interest.
+
+### TLS/HTTPS in core
+1. @leont said "I did start working on that", but hasn't shared a document yet
+2. New PSC will likely support HTTPS in core - there will be a meeting between old and new and it will likely continue
+
+### German Sovereign Tech fund is open for applications
+1. @sjn suggests a poll of who can commit time to the work that could be started based on an application
+2. Peter will help post US Perl conference
+3. We need someone who can do project management.
+
+### Secure by Default
+1. @stigtsp and @garu are putting together a list
+2. sigstore and tuf could/should be on the list
+
+### Eclipse ORC WG
+1. @sjn - goal is to give feed back on the EU standards and regulations
+2. Project has a technical lead
+3. PRF is a foundation perl noc hosts code in theory neither can join the organization - need to do both.
+
+### CycloneDX 1.7 Sustainability fields
+1. @sjn - talked to someone who is working in the sustainability of opensource projects
+
+### CPAN Meta Requirements and PURLs
+1. @sjn - no update
+
+### POSIX::2008 vulnerabilities
+1. Need to look at register CVE for that
+
+### SBOM/Supply Chain
+- @sjn - has been sharing and still a WIP.  The contents are in a good place at this time.
+- Please read and provide feedback if you can
+- Discussed at the OpenSSF numerous times
+
+## Upcoming Events
+1. ~~[The Perl and Raku Conference](https://tprc.us/tprc-2024-las/) in Las Vegas, NV - June 24-28, 2024~~
+1. [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) in Vienna, Austria - September 16-18 + 19-20 - Lots of people from OpenSSF + SBOM + Supply chain security communities
+1. [London Perl Workshop](https://act.yapc.eu/lpw2024/) - Saturday 26th October 2024 - possible talk opportunities
+
+## Other topics
+1. @Tux and @Tim worked on Crypt::OpenSSL::PKCS12
+2. @stigsto how should a secure random module get moved to core?  @garu explainde the process for getting things moved into core - requests for comment, etc.
+
+## Next meeting
+1. July 18, 2024 at 17:00 UTC
+https://www.timeanddate.com/worldclock/converter.html?iso=20240718T170000&p1=1440&p2=1129&p3=136&p4=195&p5=16&p6=187&p7=233&p8=37&p9=250&p10=234&p11=256&p12=248
```

Comment:
Remove links like these from the minutes

Reply:
We can but I am not sure that it would not be useful if we ever get timely at posting these minutes